Project3/index.html

```diff
@@ -0,0 +1,118 @@
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<meta http-equiv="X-UA-Compatible" content="ie=edge">
+<link rel="stylesheet" href="styles.css">
+<link href="https://fonts.googleapis.com/css?family=BenchNine:300,400,700|Teko:300,400,500,600,700&display=swap"
+    rel="stylesheet">
+<title>Document</title>
+</head>
+
+<body>
+
+    <form action="">
+        <h2>IDENTITY</h2>
+        <label for="real-name">Real Name</label>
+        <input id="real-name" type="text" required>
+
+        <label for="alias"> Alias </label>
+        <input id="alias" type="text" required>
+
+        <label for="email"> Email </label>
+        <input id="email" type="text" required>
+
+        <h2>PERSONAL<br> CHARACTERISTICS</h2>
```

Comment:
1. Same note about ALL CAPS.
1. The `br` element should not be used to force linebreaks for aesthetic reasons -- only meaningful ones, like in song lyrics and poetry.